src/main/java/school/lemon/changerequest/java/io/pr2/JsonString.java

```diff
@@ -0,0 +1,24 @@
+package school.lemon.changerequest.java.io.pr2;
+
+/**
+ * Created by lera on 10.02.17.
+ */
+public class JsonString implements JsonType {
+
+    private final String value;
+
+    public JsonString(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
+
+    @Override
+    public String toStringRepresentation() {
+        if ("".equals(value)) return "null";
```

Comment:
It's not correct implementation.
If value is just **empty** string - it should be saved in the same way as other strings - wrap empty string in quotes.

**BUT** if value is `null` - you must return `null` and you must not wrap anything in quotes.